index.html

```diff
@@ -3,38 +3,37 @@
   <head>
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>TV Show Project |Donara Blanc (donarbl)</title>
+    <title>TV Show Project | Donara Blanc (donarbl)</title>
     <link href="style.css" rel="stylesheet" />
     <link rel="icon" href="/favicon.ico" type="image/x-icon" />
   </head>
-
   <body>
-
     <header>
       <div class="container">
-          <h1>TV Series Episodes</h1>
-          <input type="text" id="search-box" placeholder="Search episodes..." />
+        <h1>TV Series Shows and Episodes</h1>
+        <input type="text" id="show-search" placeholder="Search shows..." />
+        <select id="show-selector">
+          <option value="">Select a Show</option>
+        </select>
+        <div id="episode-controls" style="display: none;">
+          <input type="text" id="episode-search" placeholder="Search episodes..." />
```

Comment:
These inputs do not have associated labels, so this front end does not score 100 on Lighthouse. Please do always associate form controls with labels so your interface can function correctly.